### PR TITLE
worked on mailer functionality

### PR DESCRIPTION
**Pull request by rishabhharod**

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -25,20 +27,19 @@ def create
   end
 
   def sellerProduct
-      @sellerProduct = current_user.seller.products.all
+    @sellerProduct = current_user.seller.products.all
```
Comment (aadi163) [inline app/controllers/products_controller.rb]: You are passing all products so variable name should be @seller_products.

File: app/controllers/sellers_controller.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class SellersController < ApplicationController
   def new
     @seller = Seller.new
   end
+
   def create
     if current_user.role.eql?('seller') 
       if current_user.seller.present?
```
Comment (aadi163) [inline app/controllers/sellers_controller.rb:8]: @seller = Seller.new(seller_params)
@seller.save
Why are you using  instance variable ? If there is no need to write instance variable . you can use local variable

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (rishabhharod) [reply app/controllers/products_controller.rb]: thank you for notify me i will update it

File: app/controllers/sellers_controller.rb (hunk 2 above)
Comment (rishabhharod) [reply app/controllers/sellers_controller.rb:8]: sure

Comment (rishabhharod) [reply app/controllers/sellers_controller.rb:8]: lot of factors depend upon the new and create action, during the new record is generated we have to use @instance variable in new and create action. it's also being accessible for view here is the reference https://guides.rubyonrails.org/getting_started.html

**Pull request merged**